README.md

```diff
@@ -3,14 +3,19 @@
 
 # PortFolio Website Mobile Version
 
-> This project delineates the development of portfolio.
+> This project delineates the development of my Portfolio.
 
 
 ## Built With
 
 >- HTML
 >- CSS
 
+## Live Demo (if available)
```

Comment:
- [ ] [Optional] It would be better if you remove `(if available)` because now you already have a link 😉.

Reply:
@Omar-Muhamad Great review. I didnt see that. Thanks